package.nls.json

```
@@ -391,7 +391,7 @@
     "AWS.sam.debugger.noLaunchJson": "AWS SAM: To debug a Lambda locally, create a launch.json from the Run panel, then select a configuration.",
     "AWS.sam.debugger.noTemplates": "No SAM templates found in workspace",
     "AWS.sam.debugger.noWorkspace": "AWS SAM debug: choose a workspace, then try again",
-    "AWS.sam.debugger.failedLaunch": "AWS SAM failed to launch. Try creating launch.json",
+    "AWS.sam.debugger.failedLaunch.missingRuntime": "Toolkit could not assume a runtime for config: {0}. Add a \"lambda.runtime\" field to your launch configuration.",
```

"Infer" is probably more conventional wording that "assume"

```suggestion
    "AWS.sam.debugger.failedLaunch.missingRuntime": "Toolkit could not infer a runtime for config: {0}. Add a \"lambda.runtime\" field to your launch configuration.",
```